style/properties/properties.mako.rs

```diff
@@ -1874,9 +1874,9 @@ impl ComputedValues {
     ///
     /// Usage example:
     /// let top_color =
-    ///   style.resolve_color(style.get_border().clone_border_top_color());
+    ///   style.resolve_color(&style.get_border().clone_border_top_color());
```

Comment:
For Servo this would be a better example:
```rs
style.resolve_color(&style.get_border().border_top_color);
```
but that doesn't work in Gecko so I guess it's better to leave it as is.